api/src/pcapi/routes/backoffice_v3/individual_bookings.py

```diff
@@ -329,31 +343,31 @@ def get_batch_cancel_individual_bookings_form() -> utils.BackofficeResponse:
 @individual_bookings_blueprint.route("/batch-cancel", methods=["POST"])
 @utils.permission_required(perm_models.Permissions.MANAGE_BOOKINGS)
 def batch_cancel_individual_bookings() -> utils.BackofficeResponse:
-    return _batch_individual_bookings_action(
-        success_message="Les réservations ont été annulées avec succès", validate=False
-    )
-
-
-def _batch_individual_bookings_action(success_message: str, validate: bool = False) -> utils.BackofficeResponse:
-    form = empty_forms.BatchForm()
+    form = individual_booking_forms.BatchCancelBookingForm()
     if not form.validate():
         flash(utils.build_form_error_msg(form), "warning")
         return _redirect_after_individual_booking_action()
 
+    return _batch_individual_bookings_action(
+        form,
+        lambda booking: bookings_api.mark_as_cancelled(
+            booking, bookings_models.BookingCancellationReasons(form.reason.data)
+        ),
+        "Les réservations ont été annulées avec succès",
+    )
+
+
+def _batch_individual_bookings_action(
+    form: empty_forms.BatchForm, booking_cb: typing.Callable, success_message: str
+) -> utils.BackofficeResponse:
     bookings = bookings_models.Booking.query.filter(bookings_models.Booking.id.in_(form.object_ids_list)).all()
-    api_function = bookings_api.mark_as_cancelled
-    if validate:
-        api_function = bookings_api.mark_as_used
 
     for booking in bookings:
         try:
-            api_function(booking)
+            booking_cb(booking)
```

Comment:
```suggestion
            _booking_cb(booking)
```
non ?
D'ailleurs je ne trouve pas le nom très clair (je ne devine pas ce qui est derrière le cb 😅 )

Reply:
Non, pas d'underscore, c'est le paramètre ; je renomme en `booking_callback`